### PR TITLE
Add physical plan to the dot file as the graph label

### PR DESCRIPTION
**Pull request by gerashegalov**
Inspired by #2636

Signed-off-by: Gera Shegalov <gera@apache.org>

### CODE REVIEW

File: tools/src/main/scala/com/nvidia/spark/rapids/tool/ToolTextFileWriter.scala [hunk 1]
```diff
@@ -25,8 +25,7 @@ import org.apache.spark.sql.rapids.tool.profiling._
 /**
  * Class for writing local files, allows writing to distributed file systems.
  */
-class ToolTextFileWriter(finalOutputDir: String, logFileName: String) extends Logging {
-
+case class ToolTextFileWriter(finalOutputDir: String, logFileName: String) extends Logging {
```
Comment (tgravescs) [inline tools/src/main/scala/com/nvidia/spark/rapids/tool/ToolTextFileWriter.scala:28]: why this change, please remove

Comment (gerashegalov) [reply tools/src/main/scala/com/nvidia/spark/rapids/tool/ToolTextFileWriter.scala:28]: I made this change as a shortcut to get the app id  into the label https://github.com/NVIDIA/spark-rapids/pull/2640/files#diff-8dc61637872038b82e8dda72007efa65d71ff93aae90b312c7185abd320dbe87R175

Comment (tgravescs) [reply tools/src/main/scala/com/nvidia/spark/rapids/tool/ToolTextFileWriter.scala:28]: Yeah I'd much rather not reuse this and have the app id passed in if we want it there.  CollectInformation can pass it into generateDotGraph

**Comment by gerashegalov** [comment]
build

**Comment by nartal1** [comment]
Could you please target this to branch-21.06.

**Comment by gerashegalov** [comment]
Screenshot from 2021-06-08 15-05-20

**Comment by tgravescs** [comment]
build

**Comment by gerashegalov** [comment]
looking into test failure

**Comment by gerashegalov** [comment]
build

**Review by nartal1 — APPROVED** [review]
LGTM
